### DISCUSSION TITLE
Password retrieval function

### DISCUSSION THREAD

**Discussion by JosephNew** [Difegue/LANraragi#1003] [Q&A] [answered]
I forgot my password and after many attempts, I can't find it.
I haven't found any way to get him back, and now I can't continue adding files to my media library.
I spent a lot of time uploading the existing materials, and I don't want to give them up

**Comment by Difegue** [comment, marked as answer]
Is this on Windows?  
Start LRR through the tray app, start a terminal instance on the side and enter the following to disable password auth:  
```
wsl.exe -d lanraragi
redis-cli
HSET LRR_CONFIG enablepass 0
```

**Reply by domonnss** [reply]
`redis-cli -n 2` is right,not redis-cli

**Reply by Difegue** [reply]
right, that changed when config was moved to another database within redis. Thanks!
